03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,36 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>.
+    <header>
+        <h1>The Daily Planet</h1>
+    </header>
+    <nav>
+        <a href="http://espn.go.com/">Sports</a>
+        <a href="https://weather.com/">Weather</a>
+        <a href="http://www.austinchronicle.com/">Local</a>
+    </nav>
+    <section>
+        <h1>Trump withhold support</h1>
+        <img width="100%" src="http://i2.cdn.turner.com/cnnnext/dam/assets/160114170540-ryan-trump-mccain-split-overlay-tease.jpg">
+        <p><strong>Washington (CNN)-</strong>Donald Trump is declining to endorse a pair of Republican heavyweights -- House Speaker Paul Ryan and former presidential nominee John McCain -- in their primary fights, another sign of the fractures within the
+            party.
+        </p>
+        <p>Ryan and McCain face long-shot challengers in their respective primary races in Wisconsin and Arizona. Yet their party's presidential nominee would not bless them in an interview with The Washington Post on Tuesday.</p>
+        <p>"I like Paul, but these are horrible times for our country. We need very strong leadership. We need very, very strong leadership. And I'm just not quite there yet. I'm not quite there yet," Trump said of Ryan.</p>
+    </section>
+
+    <aside>
+        <img width="100%" src="http://orig00.deviantart.net/9f66/f/2009/213/4/c/long_cat_is_long_by_plastik_rainbows.jpg" </aside>
+        <footer>
+            <a href="https://twitter.com/" </a><img src="https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcTQlqvZjBXQxqgOCS3u04TP0UvpmpIIQjfTe4VEBhWDgBMwMDIe2k67gg"/>
+        </footer>
         <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+</body>
+
+</html>

```

Comment:
Thanks for the pointer!
Making corrections now

Thanks
Clint

On Thu, Aug 4, 2016 at 4:02 PM, Ethan Stenis <notifications@github.com>
wrote:

> In 03LessonThree/in-class-project/daily-planet.html
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/86#discussion_r73602105>
> :
>
> > \ No newline at end of file
> > +</body>
> > +
> > +</html>
>
> Overall, you show a very solid grasp of design and layout here. Just fix
> the two changes above, and you are good to go!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/86/files/dca6e78d5238b5dfd95a586b5c2dbfd10f493c0e..f4f30a176afc41c1d9278fb87c0616be4d91d1ce#r73602105>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATsR8lgfMqDDPM6Ode1hMTXOZQ2Jl6pGks5qclNRgaJpZM4JVvKV>
> .
>